sdk/core/azure-core/test/ut/transport_adapter_base_test.cpp

```diff
@@ -407,7 +394,7 @@ namespace Azure { namespace Core { namespace Test {
     Azure::Core::Url hostPath(AzureSdkHttpbinServer::Delay() + "/2"); // 2 seconds delay on server
     for (int i = 0; i < 10; i++)
     {
-      Azure::Core::Context cancelThis = Azure::Core::Context::ApplicationContext.WithDeadline(
+      Azure::Core::Context cancelThis = Context{}.WithDeadline(
```

Comment:
This feels odd to new up a context to then create a new context. Is there magic C++ happening under the covers to avoid the extra allocation?

Reply:
This is a leftover from when I had deprecated ApplicationContext - it's not a pattern that I would expect to see anywhere but tests.

I can update the change to move it back to use ApplicationContext.